titus-server-master/src/main/java/com/netflix/titus/master/mesos/kubeapiserver/KubeApiServerIntegrator.java

```diff
@@ -259,6 +278,36 @@ public void launchTasks(List<TaskInfoRequest> requests, List<VirtualMachineLease
         }
     }
 
+    private void launchTasksAsync(List<TaskInfoRequest> requests) {
```

Comment:
add docs explaining that this method will still block until all requests are finished (so we still expect the Fenzo thread to be blocked while all requests are in-flight, but requests will be executed concurrently)